OpenRA.Game/Renderer.cs

```diff
@@ -162,6 +196,15 @@ public void SetPalette(HardwarePalette palette)
 		public void EndFrame(IInputHandler inputHandler)
 		{
 			Flush();
+
+			screenBuffer.Unbind();
+
+			// Render the compositor buffer to the screen
+			// HACK / PERF: Fudge the coordinates to cover the actual window while keeping the buffer viewport parameters
+			// This saves us two redundant (and expensive) SetViewportParams each frame
+			RgbaSpriteRenderer.DrawSprite(screenSprite, new float3(0, lastBufferSize.Height, 0), new float3(lastBufferSize.Width, -lastBufferSize.Height, 0));
```

Comment:
You could use BlitFramebuffer here to avoid the viewport nonsense

Reply:
`glBlitNamedFramebuffer` doesn't seem to be available from `EXT_framebuffer_object`, so this will have to wait until/after #17010